kubernetes-addons/Observability/README.md

```diff
@@ -0,0 +1,137 @@
+# How-To Setup Observability for OPEA Workload in Kubernetes
+
+This guide provides a step-by-step approach to setting up observability for the OPEA workload in a Kubernetes environment. We will cover the setup of Prometheus and Grafana, as well as the collection of metrics for Gaudi hardware, OPEA/chatqna including TGI,TEI-Embedding,TEI-Reranking and other microservies, and PCM.
+
+## 1. Setup Prometheus & Grafana
+
+Setting up Prometheus and Grafana is essential for monitoring and visualizing your workloads. Follow these steps to get started:
+
+### Step 1: Install Prometheus&Grafana
+
+```
+kubectl create ns monitoring
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+helm install prometheus-stack prometheus-community/kube-prometheus-stack --version 55.5.1 -n monitoring
+```
+
+### Step 2: Verify the installation:
+
+```
+kubectl get pods -n monitoring
```

Comment:
```suggestion
kubectl -n monitoring get all | grep -e NAME -e prom
```

Reply:
You can observe HTTP request latency, which is useful for benchmarking, as well as for analyzing and identifying the bottlenecks in all the microservices.

Reply:
Ok, I see.  As you answered to a different comment, I'll edit the comment this actually concerns based on your answer (as this will get hidden when it's resolved).

Reply:
This is an lightweight addon for monitor the metrics of all the services in chatQnA, no need to touch or modify the exist helm-chart deployment, and no need for end-user to enable HPA. If a user wants to see the metrics in grafana he could setup this addon and he could also uninstall it after a while.